packages/sucrase/src/index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+
+const { transform } = require('sucrase');
+const { createFilter } = require('rollup-pluginutils');
```

Comment:
@rollup/pluginutils - I'm not sure how tests passed because you've got the scoped package in package.json

Reply:
Interesting, other packages in the repo still have `rollup-pluginutils` in their deps so it probably used that from the top level `node_modules`.